conf/eslint-all.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+var lodash = require("lodash");
+var fs = require("fs");
+var path = require("path");
+
+var ruleFiles = fs.readdirSync(path.resolve(__dirname, "../lib/rules"));
```

Comment:
I think we should lazyload this since not everyone is going to be using `eslint:all` but they would still endup paying the price for a whole directory scan at load time.